### DISCUSSION TITLE
Upcoming Washer Support

### DISCUSSION THREAD

**Discussion by QuarterSwede** [nVuln/homebridge-lg-thinq#3] [General]
Definitely interested in the upcoming washer support. We just got an LG top load washer with ThinQ and I’d be willing to help test it. As an aside, we do not have a ThinQ dryer.

**Comment by nVuln** [comment]
I'm currently working on my LG washer & dryer combo, it will come together 🙋‍♂️

**Reply by markymark125** [reply]
Hi

I just got the 	
LG V11 FWV1128WTSA And would be willing to test anything to get more integration with HK

**Comment by nVuln** [comment]
@QuarterSwede I already added Washer, please try if it work for you

**Reply by QuarterSwede** [reply]
It shows up in HK just fine. It shows the following statuses: On and waiting to start, child lock, and door lock (should be called lid for my top load model). Even though the statuses show as a switch to turn on and off you can’t do anything with the switch. I understand that LG doesn’t support that because of safety for children and pets. Just wondering if there is a better way to show its status that doesn’t make the user think they can control it from HK. Hope that makes sense.

It also shows the water temp. 

I haven’t tried washing anything yet because I don’t have a load to do but once I use it I’ll let you know if notifications work etc.

**Reply by nVuln** [reply]
I think we should disable Door Lock and Child Look status until Homekit official support Washer, for now running status is enough

**Reply by TallWilber** [reply]
I was able to add my washer & dryer. Is there a way to enable to time remaining to show up on the HomeKit tile itself? Right now I need to long press to see it.

**Comment by QuarterSwede** [comment]
Finished a bunch of loads yesterday and while the LG ThinQ app notifies the user it doesn’t through HomeKit. The washer doesn’t show up in the HomeKit notification management area either.

**Reply by nVuln** [reply]
atm we can't forward notification from LG app to homekit
in next release I added event trigger when washer finished, you need manual setup automation on homekit to capture this

**Reply by jtauke** [reply]
Is there a best way to hook into the washer finished event?  Can't seem to figure out how to use it to fire a notification.

**Reply by nVuln** [reply]
you can try [GetPushed](https://pushed.co/)
AppStore: https://apps.apple.com/us/app/get-pushed/id804777699
Homebridge plugin: https://www.npmjs.com/package/homebridge-pushed-notification

**Comment by batkoto** [comment]


**Reply by nVuln** [reply]
> Is the current plugin integration to show as VALVE until the Washing Machine support is officially supported in HomeKit ?
> Just making sure I had the setup correct as I am new to the Homebridge integrations
> 
> Apologies if this was discussed previously

correct.

**Comment by mdisabato** [comment]
Hi!

We just purchased a WashTower. I do not see it in the bridge. If this is something you could work into the plugin, I have a debug run I can send. 

Thank you.

Michael

**Reply by nVuln** [reply]
> Hi!
> 
> We just purchased a WashTower. I do not see it in the bridge. If this is something you could work into the plugin, I have a debug run I can send.
> 
> Thank you.
> 
> Michael

yes, please open issue and upload debug log, let me see

**Comment by JoelGit** [comment]
May I ask the status of washer and dryer support? Just picked up a 7300 and 7305 models and wondering if this plugin will work.

**Reply by nVuln** [reply]
> May I ask the status of washer and dryer support? Just picked up a 7300 and 7305 models and wondering if this plugin will work.

washer & dryer is currently supported (both thinq1 and thinq2), but showing status only (door lock status, running status, remain duration), control feature may added later (still working on it)
install plugin and try it,  open issue if you got any problem

**Reply by JoelGit** [reply]
Will do. Thanks much for the reply!

**Comment by JoelGit** [comment]
Long time coming, but finally got this set up. In the Home app I can see my washer and dryer. When I tap and hold the dryer, I can see the countdown of remaining time. Is there any way to ask Siri to tell me the remaining time?

**Reply by nVuln** [reply]
try Siri shortcuts: https://support.apple.com/en-vn/guide/shortcuts/welcome/ios
